src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -1843,27 +1818,13 @@ public function testPhpConstantTagMappingKey()
         $this->assertSame($expected, $this->parser->parse($yaml, Yaml::PARSE_CONSTANT));
     }
 
-    public function testPhpConstantTagMappingKeyWithKeysCastToStrings()
```

Comment:
I don't remember why i removed it so I guess we can revert this. However I won't be able to do it for the next 2 weeks, feel free to push on my repository if you have the time, otherwise I'll do the change ASAP.